src/layout/internal/LineUpLayoutContainer.ts

```diff
@@ -1,16 +1,23 @@
 import {ILayoutContainer, ILayoutDump} from '../interfaces';
 import {EOrientation, IDropArea} from './interfaces';
-import {ALayoutContainer} from './ALayoutContainer';
+import {ALayoutContainer, ILayoutContainerOption} from './ALayoutContainer';
 import {ASequentialLayoutContainer, ISequentialLayoutContainerOptions, wrap} from './ASequentialLayoutContainer';
 
+export interface ILineUpLayoutContainer extends ISequentialLayoutContainerOptions {
+  stackLayout?: boolean;
```

Comment:
why optional, since already Partial is used later one?

Reply:
and since this is a new option, please extends also `builder.ts` accordingly and the dumping and restoring methods